### ISSUE TITLE
[A11yInspect] Accessibility Report - 43 fails

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#7]
## Accessibility Report

**Generated:** 2026-02-23T06:40:48.039Z

### Summary
- ❌ **Fails:** 43
- **Files checked:** 3

> Validates, Suggestions, and Passes are excluded from issue tracking.

### Fail Issues

- [x] #8
- [x] #9
- [ ] #10
- [ ] #11
- [ ] #12
- [ ] #13
- [ ] #14
- [ ] #15
- [ ] #13
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #9
- [ ] #12
- [ ] #16
- [ ] #17
- [ ] #18
- [ ] #19
- [ ] #20
- [ ] #21
- [ ] #22
- [ ] #23
- [ ] #24
- [ ] #25
- [ ] #23
- [ ] #22
- [ ] #26
- [ ] #22
- [ ] #27
- [ ] #22
- [ ] #22
- [ ] #22
- [ ] #22
- [ ] #22
- [ ] #22
- [ ] #22
- [ ] #25

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
